pkg/runner/local_runner.go

```diff
@@ -166,6 +225,14 @@ func (r *localRunner) Run(ctx context.Context, request *runner.RunRequest) (*run
 		}
 	}
 
+	if requiresSpecialFilesystems {
```

Comment:
This is not defered, as I saw some unmounting errors during development that would not surface well.
Do you want me to create a defer-lambda that modifes the `err` on the way out to set an error
or is this an okay clean-up section?

Reply:
Yeah, having it as a `defer func() {}()` makes sense.